cogboard-webapp/src/components/widgets/types/AemHealthcheckWidget/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { string, object } from "prop-types";
+
+import { Link, Typography } from '@material-ui/core';
+import { Caption } from "../../../styled";
+
+import { AEM_HEALTH_CHECKS } from '../../../../constants';
+
+const AemHealthcheckWidget = props => {
+  const { healthChecks, errorMessage } = props;
+
+  if (errorMessage) {
```

Comment:
I have noticed that it was used couple of times, maybe it is the highest time to extract it to it's own component like ErrorMessage.

Reply:
I'm not convinced

Reply:
Discussed.